packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/updateSelections/bringFineAfterThis.yml

```diff
@@ -19,7 +19,7 @@ initialState:
       start: {line: 0, character: 0}
       end: {line: 0, character: 3}
 finalState:
-  documentContents: foo  fooworld
+  documentContents: foo world foo
```

Comment:
Oh wow how did we miss this? Definitely seems like an improvement 👍

Reply:
...but no longer really testing what we wanted it to.  I'll move this test somewhere else to indicate it is about "this" expansion and re-record these tests with "just"

Reply:
One thing that is a bit confusing now is that "bring air to end of this" won't expand to token, but "bring air after this" will 🤔.  I'm happy to call it out of scope for this PR, but maybe I should file a follow up to discuss?